src/test/java/org/jalasoft/moi/model/python/PythonCommandBuilderTest.java

```diff
@@ -31,12 +31,13 @@
 
     @BeforeAll
     static void initAll() {
-        String python = "\\thirdparty\\python\\win\\python32\\Portable Python 3.2.5.1\\App\\python.exe";
+        String actualParentPath = System.getProperty("user.dir");
+        String python = actualParentPath + "\\thirdparty\\python\\win\\python32\\Portable_Python_3.2.5.1\\App\\python.exe";
```

Comment:
Here is another example where `System.getProperty("user.dir");` is needed, it can become a global constant.